server/routes/payments.js

```diff
@@ -4,9 +4,9 @@ const express = require('express');
 const router = express.Router();
 const PaymentCard = require('../database/models/PaymentCard');
 const isLoggedInGuard = require('../middleware/isLoggedInGuard');
-const itemOwnerGuard = require('../middleware/itemOwnerGuard');
+const paymentCardGuard = require('../middleware/paymentGuard');
 
-router.route('/').get(isLoggedInGuard, (req, res) => {
+router.route('/').get(isLoggedInGuard, paymentCardGuard, (req, res) => {
```

Comment:
we do need the guard, but just not on the get.  Leave it in the file because I'll add it to the put/delete